addOns/alertFilters/src/main/java/org/zaproxy/zap/extension/alertFilters/automation/AlertFilterJobDialog.java

```diff
@@ -0,0 +1,209 @@
+                                        job,
+                                        getAlertFilterModel(),
+                                        getAlertFilterModel().getAlertFilters().get(row),
+                                        row);
+                        dialog.setVisible(true);
+                    });
+        }
+        return this.modifyButton;
+    }
+
+    private JButton getRemoveButton() {
+        if (this.removeButton == null) {
+            this.removeButton =
+                    new JButton(Constant.messages.getString("automation.dialog.button.remove"));
+            this.removeButton.setEnabled(false);
+            final AlertFilterJobDialog parent = this;
+            this.removeButton.addActionListener(
+                    e -> {
+                        if (JOptionPane.OK_OPTION
+                                == View.getSingleton()
+                                        .showConfirmDialog(
+                                                parent,
+                                                Constant.messages.getString(
+                                                        "automation.dialog.requestor.remove.confirm"))) {
+                            getAlertFilterModel().remove(getAlertFilterTable().getSelectedRow());
+                        }
+                    });
+        }
+        return this.removeButton;
+    }
+
+    private JTable getAlertFilterTable() {
+        if (alertFilterTable == null) {
+            alertFilterTable = new JTable();
+            alertFilterTable.setModel(getAlertFilterModel());
+            alertFilterTable
+                    .getColumnModel()
+                    .getColumn(0)
+                    .setPreferredWidth(DisplayUtils.getScaledSize(200));
+            alertFilterTable
+                    .getColumnModel()
+                    .getColumn(1)
+                    .setPreferredWidth(DisplayUtils.getScaledSize(80));
+            alertFilterTable
+                    .getColumnModel()
+                    .getColumn(2)
+                    .setPreferredWidth(DisplayUtils.getScaledSize(120));
+            alertFilterTable
+                    .getSelectionModel()
+                    .addListSelectionListener(
+                            e -> {
+                                if (getAlertFilterTable().getSelectedRowCount() == 0) {
+                                    modifyButton.setEnabled(false);
+                                    removeButton.setEnabled(false);
+                                } else if (getAlertFilterTable().getSelectedRowCount() == 1) {
+                                    modifyButton.setEnabled(true);
+                                    removeButton.setEnabled(true);
+                                } else {
+                                    modifyButton.setEnabled(false);
+                                    removeButton.setEnabled(false);
+                                }
```

Comment:
Should do the same way as in #3076 (there might be other relevant changes). Maybe that should be merged first.